catalyst/core/callbacks/wrapper.py

```diff
@@ -8,6 +8,113 @@
 FILTER_FN = Callable[[str, int, str], bool]
 
 
+class WrapperCallback(Callback):
```

Comment:
maybe,  we should move it to `catalyst/core/callback`?

Reply:
sure